deployments/cardano-prod.nix

```diff
@@ -0,0 +1,12 @@
+with (import ./../lib.nix);
+
+let
+  conf = { config, pkgs, resources, ... }: {
+    imports = [
+      ./../modules/datadog.nix
+    ];
+    services.dd-agent.tags = ["prod"];
+  };
+in {
+  report-server = conf;
+} // (genAttrs' (range 0 13) (key: "node${toString key}") (name: conf))
```

Comment:
Good points!

There are two refactorings needed for this, and they've been sitting on the back of my mind for some time. I don't want to block this PR, as we need this soon, but we should make sure they come up quickly:

1)  make number of nodes a deployment input that can be specified in the config file and reused all over the place
2) separate EC2 specific settings into `explorer-prod.nix` and `explorer-staging.nix`, mostly based on `modules/amazon-base.nix` (this would allow us to build AMI without hacks like `generatingAMI`

I've opened DEVOPS-37 to address this, as they both touch the same code.

Concerning `node0` up to `nodeX`: these are name of the nodes in a cluster. Cluster is uniquely identified by deployment name, nodes always go from 0 to X. If you have a suggestion how to make this better (except from moving the number of nodes to a config file), I'm all ears :)